test/exceptions/Makefile

```diff
@@ -61,17 +61,19 @@ $(ROOT)/unittest_assert.done: STDERR_EXP="unittest_assert msg"
 $(ROOT)/invalid_memory_operation.done: STDERR_EXP="InvalidMemoryOperationError"
 $(ROOT)/unknown_gc.done: STDERR_EXP="'unknowngc'"
 $(ROOT)/static_dtor.done: STDERR_EXP="dtor_called_more_than_once"
-$(ROOT)/future_message.done: STDERR_EXP="exception I have a custom message. exception exception "
 $(ROOT)/static_dtor.done: NEGATE=!
```

Comment:
is this reordering intentional?

Reply:
Yes - keeping the `static_dtor.none` stuff together. `NEGATE=!` means the `STDERR_EXP[2]` must **not** be found in stderr.